### DISCUSSION TITLE
Question: What is the major difference between geffnet and pytorch-image-models

### DISCUSSION THREAD

**Discussion by ghnreigns** [rwightman/gen-efficientnet-pytorch#57] [Q&A]
Dear Ross, this is not an issue per se, but just a way to ask a possibly duplicated question. Will there be any major difference in using geffnet rather than the ones in pytorch-image-models? I noticed both use the same pre-trained weights and most of the process is the same.

**Comment by rwightman** [comment]
They are as you say essentially the same. However pytorch-image-models are a bit more integrated with other aspects of timm with my own interfaces for reseting the head, feature extraction wrappers, etc. `timm` models also have default_cfg dicts to describe the dataset mean/std and make it easy for the library to setup augmentation pipeline.

This is a more bare bones impl that is closer to what you'd find in torchvision without the extra dependencies and goodies. I've also included demo/example ONNX and Caffe2 export code here that isn't (yet) in timm.

**Comment by ghnreigns** [comment]
@rwightman Thank you Ross for the detailed reply. I will stick to `geffnet` for EfficientNets and any other models to `timm`. Really appreciate the work you have done here!
